### DISCUSSION TITLE
Getting local point position in global space is confusing

### DISCUSSION THREAD

**Discussion by JohnTheCoolingFan** [bevyengine/bevy#8501] [Q&A] [answered]
I need to take a local vector from `Transform::back` for example and recalculate it to global space (rapier's ExternalForce applied at a point requires that). I found `transform_point` in `Transform` and `GlobalTransform`, but got confused by the documentation, I don't understand what these functions do and which one I should use for my task.

**Comment by tim-blackbird** [comment, marked as answer]
`Transform::back` is not a position but a normalized vector.
You can use `global_transform.affine().transform_vector3(vector).normalize()` to transform a local normalized vector to a global one.

`GlobalTransform::transform_point` is for positions and the docs on it are currently wrong and state the opposite of what it actually does (It actually transforms the point from local to global).

For `Transform::back` you can use `GlobalTransform::back` instead.

**Reply by JohnTheCoolingFan** [reply]
I got point position an vector confused because they are both Vec3... But anyways, I do need to convert local vector to global in a system. One system sets the local vector in a component, other uses that to get a global vector for thrust and torque. Thanks for your solution (`global_transform.affine().transform_vector3(vector).normalize()`), I'll try that.
